stdpopsim/species.py

```diff
@@ -219,6 +229,29 @@ def add_demographic_model(self, model):
             )
         self.demographic_models.append(model)
 
+    def get_dfe(self, id):
+        """
+        Returns a dfe with the specified ``id``.
```

Comment:
I'd have thought we'd capitalize DFE, so `get_DFE` and `Returns a DFE`? It's an abbreviation?